README.md

```diff
@@ -14,6 +14,15 @@ Dexie solves three main issues with the native IndexedDB API:
 
 Dexie provides a neat database API with a well thought-through API design, robust error handling, extendability, change tracking awareness and extended KeyRange support (case insensitive search, set matches and OR operations).
 
+#### Features
+
+* Promise-based API towards indexedDB (supports all IDB features)
+* Works with any framework such as React or Angular2
+* Client/Server synchronization via [sync-client](https://www.npmjs.com/package/sync-client). Can adapt to custom sync backends via [dexie-syncable](https://www.npmjs.com/package/dexie-syncable)
```

Comment:
Maybe add sync-server since the synchronization needs a server. I don't know if there are other compatible servers out there. FYI sync-server also works with websockets which sync-client does not.